### PR TITLE
Fix Salinity argument in pyCRTM.py

### PR DESCRIPTION
**Pull request by ejsimon9**
S2m[:,1] is labeled as "T (2meter)", but is being passed as though it is salinity. Skin[:,1] should be used instead.

## Description
This fixes which variable is passed as the salinity value to CRTM

### CODE REVIEW

**Comment by karpob** [comment]
Ah, yeah what I did there is not the greatest. The fix is a little bigger than that as the examples would need to be updated to get rid of S2m since it wouldn't do anything.

**Comment by karpob** [comment]
@ejsimon9 could you invite me to modify your fork so I can commit changes. I have a change to make Salinity cleaner. Basically call it what it is: Salinity.  I'm guessing this is Eric from NRL?

**Comment by ejsimon9** [comment]
Hi Bryan,

I hope you're doing well (and yes, this is Eric from NRL).

Thanks for looking into this. I just invited you to modify my fork. Let me
know if you need any help.

Eric

**Comment by karpob** [comment]
@ejsimon9. Great - I'm doing well. I think the commit I just made cleans up the API to be a little less cryptic. At one point I was trying to mirror the RTTOV python interface, but there are some things that just don't translate 100% between RTTOV and CRTM, so I diverged a little bit in points, and didn't get back to cleaning things up in others. Skin/S2m was one of those things. Let me know if you think it makes sense.

**Comment by ejsimon9** [comment]
Thanks Bryan, that looks good- much more clear

**Comment by StegmannJCSDA** [comment]
I'm having a look at the PR today.

**Comment by StegmannJCSDA** [comment]
I get the following error when I run `./setup_pycrtm` on macOS:

```shell
Traceback (most recent call last):
  File "./setup_pycrtm.py", line 288, in <module>
    main(a)
  File "./setup_pycrtm.py", line 23, in main
    with open(os.path.join('libsrc','CRTM_Version.inc'),'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'libsrc/CRTM_Version.inc'
```

Specifically I was trying:
```bash
./setup_pycrtm.py  --install ${CRTM_ROOT} --repos ${CRTM_ROOT} --jproc 1 --coef $PWD --ncpath /usr/local/Cellar/netcdf/4.7.4_1 --h5path /usr/local/Cellar/hdf5/1.12.0_1 --arch gfortran-openmp
```

**Comment by karpob** [comment]
@StegmannJCSDA Yes, that is the bigger problem system wide. The master branch won't work as it still expects the Autoconf-style build. It was something that made me a bit uncomfortable during release, and I was hoping a better solution would present itself sooner than it has. 

The only one that sort-of works if you follow instructions is the crtm-bundle which has that kickstart script that points to feature/btj_crtm-bundle. I've been able to make it work on a platform other than in a container, but I need to add to LD_LIBRARY_PATH.

File: testCases/test_atms.ipynb [hunk 1]
```diff
@@ -21,7 +21,8 @@
     "\n",
     "<h3>Record of Revisions:</h3>\n",
     "    \n",
-    "Date: 2020-02-26 Author: P. Stegmann Description: Original code"
+    "Date: 2020-02-26 Author: P. Stegmann Description: Original code\\\n",
+    "&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2021-03-01 Author: B. Karpowicz Description: update API to use Salinity instead of S2m"
```
Comment (StegmannJCSDA) [inline testCases/test_atms.ipynb:25]: Can this be removed?
```
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

**Review by StegmannJCSDA — CHANGES_REQUESTED** [review]
Looks good. 
Just requesting a minor change to remove the `&nbsp` in the Jupyter notebook.

File: testCases/test_atms.ipynb (hunk 1 above)
Comment (StegmannJCSDA) [reply testCases/test_atms.ipynb:25]: What do you think of using a table?
[https://stackoverflow.com/questions/48655801/tables-in-markdown-in-jupyter](url)

Comment (karpob) [reply testCases/test_atms.ipynb:25]: I like, it. I'll put one in.

**Comment by karpob** [comment]
@StegmannJCSDA The first cell I added will align the whole table to the left. I can't for the life of me figure out how to hide that block.

**Comment by StegmannJCSDA** [comment]
@karpob When I declare the line a markdown cell in Jupyter itself it only displays `%%html` and not the entire CSS. I think that's as good as it gets.

**Comment by karpob** [comment]
@StegmannJCSDA Yeah, when I do that though, it just moves the table back to the center. Ideally, I'd like it justified to the left, but I don't think it's worth the extra effort.

**Review by StegmannJCSDA — APPROVED** [review]

**Review by karpob — APPROVED** [review]

**Pull request merged**